pages/docs/reference/classes.md

```diff
@@ -261,6 +261,21 @@ class Example // Implicitly inherits from Any
 
 `Any` has three methods: `equals()`, `hashCode()` and `toString()`. Thus, they are defined for all Kotlin classes. 
 
+#### "open" Keyword
```

Comment:
Please remove the heading. `open` is the synonym for inheritance in Kotlin, so we can introduce it right under the `Inheritance` heading.